app/views/messages/new.html.erb

```diff
@@ -0,0 +1,9 @@
+
+<%= form_for @message, url: {action: "create"}, html: {class: "nifty_form"} do |f| %>
+ <%= f.label :message %>
+  <%= f.text_field :content %>
+  <% hidden_field :message, :user_id, :value => session[:user_id] %>
+  <% hidden_field :message, :conversation_id, :value => params[:id] %>
```

Comment:
The params hash is not something accessible on the view. You should be passing this data down (`@conversation`) from your message controller's new action.

Also have a look at `hidden_field_tag` which gives you a little more configuration than plain old `hidden_field`. [Docs here.](http://apidock.com/rails/ActionView/Helpers/FormTagHelper/hidden_field_tag)